simplified-opds-core/src/main/java/org/nypl/simplified/opds/core/OPDSAcquisitionFeedEntryParser.java

```diff
@@ -284,7 +284,7 @@ private void tryConsumeAcquisitions(
     final String rel_text)
     throws OPDSParseException {
 
-    if (rel_text.startsWith(ACQUISITION_URI_PREFIX_TEXT)) {
+    if (rel_text.startsWith(ACQUISITION_URI_PREFIX_TEXT) || rel_text.toLowerCase().equals("preview")) {
```

Comment:
"preview" should probably be in a constant, because we'll probably need to use it again in other places.

Reply:
This shouldn't be here, sorry! I added this in order to reproduce the error with the latest changes from the server and it ended up here by mistake.

Reply:
Ok, thanks.